superset-frontend/src/views/CRUD/data/query/QueryList.tsx

```diff
@@ -181,12 +181,9 @@ function QueryList({ addDangerToast, addSuccessToast }: QueryListProps) {
         accessor: QueryObjectColumns.start_time,
         Header: t('Time'),
         size: 'xl',
-        Cell: ({
-          row: {
-            original: { start_time, end_time },
-          },
-        }: any) => {
-          const startMoment = moment.utc(start_time).local();
+        Cell: ({ row: { original } }: any) => {
+          const startTime = parseFloat(original.start_time);
```

Comment:
Why do we parse the string to a float, instead of web server response a float value directly? This issue seems that we should change the `start_time` and `end_time` from the server side to the float.

Those columns in `Query` was numberic. We should fix the response type.
https://github.com/apache/superset/blob/a6bf041eddcde0247461f35c806414df00ef105e/superset/models/sql_lab.py#L97

Reply:
Agreed, fixed.

Reply:
@diegomedina248 , I absolutely believe it works pretty well, but for just one confirmation. 
Can you please check if the date field in text report to email is formatted correctly? 
(Reason: It seems like by saving date in query result as a string, we resolved this issue in text report.)
Step:
1. Create a table chart including dates.
2. Create a report sending this chart as a report to email.
3. Date should be formatted as it appears in the chart.

The result we don't expect:
![Clipboard 2022-20-01 at 9 24 25 PM](https://user-images.githubusercontent.com/39701522/162946812-0036f949-8775-4b97-81c9-000483572e78.png)

Reply:
Yeah, as far as I can tell with my tests, this change does not affect that.
The code changes only affect the JSON response in the API, so we should be good there.